src/io/leopard/toLeopard.ts

```diff
@@ -8,10 +8,71 @@ import * as prettier from "prettier";
 import Target from "../../Target";
 import { List, Variable } from "../../Data";
 
-function uniqueNameGenerator(usedNames: string[] = []) {
+// Words which are invalid for any JavaScript identifier to be, when it isn't
+// on a namespace (like `this` or `this.vars`).
+//
+// This list may be more comprehensive than it needs to be in every case,
+// erring to avoid potential issues.
```

Comment:
It might be worth adding a comment detailing where this list of reserved words comes from (MDN? the spec?)